01/pwilson802/wordvalue.py

```diff
@@ -0,0 +1,30 @@
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words():
+    """Load dictionary into a list and return list"""
+    res = []
+    with open(DICTIONARY) as file:
+        for line in file:
+            res.append(line.strip())
+    return res
+
+def calc_word_value(word):
+    """Calculate the value of the word entered into function
+    using imported constant mapping LETTER_SCORES"""
+    score = 0
+    for letter in word:
+        if letter.upper() in LETTER_SCORES:
+            score += LETTER_SCORES[letter.upper()]
+    return score
+
+def max_word_value(words):
```

Comment:
You probably need to set default value for input parameter. 
Like this:
def max_word_value(words = load_words()):

Reply:
Thanks for the feedback.  I made another commit with that change.